README.md

```diff
@@ -1 +1,8 @@
-# create202404
\ No newline at end of file
+# create202404
+Some description that doesnt mean anything.
+
+## Section 2
+How to enable something that need to be enabled without the need to enable it.
+
+## Section 1
+Section that should be before #section2 but for some reason is after
```

Comment:
The order of sections seems incorrect. `## Section 1` is supposed to be before `## Section 2` but it's listed afterwards. Consider rearranging the sections to improve the logical flow of the document.